plugin/pyproject.toml

```diff
@@ -0,0 +1,25 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+# name = "bgutil-ytdlp-pot-provider"
+name = "definitely_not_bgutil-ytdlp-pot-provider_to_distinguish_from_it"
+version = "0.2.4"
+readme = {file = "README.md", content-type = "text/markdown"}
+requires-python = ">=3.8"
+keywords = ["yt-dlp", "yt-dlp-plugin"]
+authors = [
+  { name = "Brainicism", email = "brainicism@gmail.com" },
+]
+dependencies = []
```

Comment:
Can we add yt-dlp-get-pot as a dependency

Reply:
breaks github release workflow. trying to fix it

Reply:
fixed

Reply:
Actually I'm not sure whether this(adding yt-dlp-get-pot to the dependencies) is expected. Because the upstream(yt-dlp/yt-dlp-sample-plugins#3) meant to pack the dependencies into the bundle, but imo we don't need to pack yt-dlp-get-pot into our zip. I simply removed the part of the script that pack dependencies. It's buggy.

Reply:
My impression was just based off of: 
```
The provider plugin assumes this plugin is installed. You can define it as a Python dependency in your plugin package, or users can install it manually.
```

https://github.com/coletdjnz/yt-dlp-get-pot?tab=readme-ov-file#developing-a-provider-plugin 

Might be worth raising with cole but not a blocker for this PR

Reply:
alright let's resolve this conversation